src/app/ui/sequence.tsx

```diff
@@ -48,7 +48,15 @@ export function splitModelEntityId(modelEntityId: string) {
     return [parseInt(modelIdx), entityId];
 }
 
-export function getSequenceWrapper(state: { structure: Structure, modelEntityId: string, chainGroupId: number, operatorKey: string }, structureSelection: StructureSelectionManager): SequenceWrapper.Any | string {
```

Comment:
It looks like this file was prettified, so the diff is much larger than the real one. Can you restart the file and apply the changes again, without prettification?

Reply:
@tokland I actually "prettified" by myself because of the comment about keeping the line length around 140 characters, so I prettified every line that went over that length... now I see that.. that comment was only for our own modifications right?

Reply:
@p3rcypj Yes, this applies only to our new code. The ultimate goal is to ensure new code is readable while adding no diffs in the old code.